stan/math/prim/arr/functor/coupled_ode_observer.hpp

```diff
@@ -1,11 +1,9 @@
 #ifndef STAN_MATH_PRIM_ARR_FUNCTOR_COUPLED_ODE_OBSERVER_HPP
 #define STAN_MATH_PRIM_ARR_FUNCTOR_COUPLED_ODE_OBSERVER_HPP
 
+#include <stan/math/prim/meta.hpp>
 #include <stan/math/prim/scal/err/check_size_match.hpp>
 #include <stan/math/prim/scal/err/check_less.hpp>
-#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
-#include <stan/math/prim/scal/meta/operands_and_partials.hpp>
-#include <stan/math/prim/scal/meta/broadcast_array.hpp>
 #include <stan/math/prim/arr/fun/sum.hpp>
 #include <stan/math/prim/scal/meta/return_type.hpp>
```

Comment:
This file still has a meta include: `#include <stan/math/prim/scal/meta/return_type.hpp>`.  This should be removed.